docs/local-dev/cli-build-system.md

```diff
@@ -595,6 +595,19 @@ If you also want to enable source maps when debugging tests, you can do so by se
 }
 ```
 
+### Debugging Jest Tests
```

Comment:
I think we need to fold in some of the content above under this header. Perhaps we can have separate VSCode and IntelliJ sections

Reply:
No I mean the last bit of the entire section above is about how to configure debugging in VS Code 😅, starting with "If you want to configure editor integration for tests we...". What I'm looking for is just to have the "### Debugging Jest Tests" moved up above that, and then also put the stuff above under "#### Visual Studio Code"

Reply:
Ah ok, i'm sorry.
Adjusted the text, as now ENABLE_SOURCE_MAPS is not required to be true, source maps are always inlined.